src/roswire/ros2/state.py

```diff
@@ -75,6 +105,9 @@ def probe(self) -> ROS2SystemState:
             "pub": {},
             "sub": {},
             "serv": {},
+            "cli": {},
```

Comment:
It might help readability (and avoid some string bugs, see below) by changing these shorthand names to longer form equivalents (e.g., `service_clients`, `publishers`, `subscribers`, `service_providers`, etc.).

Reply:
See above